Test failure with bleak 0.20+

**Describe the bug**
API changes in bleak 0.20 make the test suite fail against pysnooz 0.8.3

The general failure looks like this:

```
>       device = BLEDevice("AA:BB:CC:DD:EE:FF", "Snooz-EEFF")
E       TypeError: BLEDevice.__init__() missing 2 required positional arguments: 'details' and 'rssi'
```

**To Reproduce**
Upgrade bleak to the version used in home-assistant and run the testsuite

**Additional context**

```
ERROR tests/test_api.py::test_properties - TypeError: BLEDevice.__init__() missing 2 required positional arguments: 'd...
ERROR tests/test_api.py::test_retries_write_errors - TypeError: BLEDevice.__init__() missing 2 required positional arguments: 'd...
ERROR tests/test_api.py::test_raises_write_errors_after_retries_exhausted - TypeError: BLEDevice.__init__() missing 2 required positional arguments: 'd...
ERROR tests/test_api.py::test_raises_unknown_write_errors - TypeError: BLEDevice.__init__() missing 2 required positional arguments: 'd...
ERROR tests/test_device.py::test_display_name - TypeError: BLEDevice.__init__() missing 2 required positional arguments: 'd...
ERROR tests/test_device.py::test_basic_commands - TypeError: BLEDevice.__init__() missing 2 required positional arguments: 'd...
ERROR tests/test_device.py::test_auto_reconnect - TypeError: BLEDevice.__init__() missing 2 required positional arguments: 'd...
ERROR tests/test_device.py::test_auto_reconnect_device_unavailable - TypeError: BLEDevice.__init__() missing 2 required positional arguments: 'd...
ERROR tests/test_device.py::test_manual_disconnect - TypeError: BLEDevice.__init__() missing 2 required positional arguments: 'd...
ERROR tests/test_device.py::test_establish_connection_exceptions - TypeError: BLEDevice.__init__() missing 2 required positional arguments: 'd...
ERROR tests/test_device.py::test_write_exception_during_reconnection - TypeError: BLEDevice.__init__() missing 2 required positional arguments: 'd...
ERROR tests/test_device.py::test_manual_disconnect_during_reconnect - TypeError: BLEDevice.__init__() missing 2 required positional arguments: 'd...
ERROR tests/test_device.py::test_disconnect_before_ready - TypeError: BLEDevice.__init__() missing 2 required positional arguments: 'd...
ERROR tests/test_device.py::test_unexpected_error_before_ready - TypeError: BLEDevice.__init__() missing 2 required positional arguments: 'd...
ERROR tests/test_device.py::test_unexpected_error_during_execution - TypeError: BLEDevice.__init__() missing 2 required positional arguments: 'd...
ERROR tests/test_device.py::test_disconnect_before_ready_then_reconnects - TypeError: BLEDevice.__init__() missing 2 required positional arguments: 'd...
ERROR tests/test_device.py::test_manual_disconnect_before_ready - TypeError: BLEDevice.__init__() missing 2 required positional arguments: 'd...
ERROR tests/test_device.py::test_disconnect_while_reconnecting_before_ready - TypeError: BLEDevice.__init__() missing 2 required positional arguments: 'd...
ERROR tests/test_device.py::test_device_disconnects_during_transition - TypeError: BLEDevice.__init__() missing 2 required positional arguments: 'd...
ERROR tests/test_device.py::test_user_disconnects_during_transition - TypeError: BLEDevice.__init__() missing 2 required positional arguments: 'd...
ERROR tests/test_device.py::test_device_unavailable_during_transition - TypeError: BLEDevice.__init__() missing 2 required positional arguments: 'd...
ERROR tests/test_device.py::test_manual_disconnect_during_transition - TypeError: BLEDevice.__init__() missing 2 required positional arguments: 'd...
ERROR tests/test_device.py::test_new_commands_cancel_existing - TypeError: BLEDevice.__init__() missing 2 required positional arguments: 'd...

--- a/tests/test_api.py
+++ b/tests/test_api.py
@@ -21,7 +21,7 @@ DBUS_ERROR_IN_PROGRESS = BleakDBusError("org.bluez.Error.InProgress", [])
 
 @pytest.fixture()
 def mock_client() -> Generator[MockSnoozClient, None, None]:
-    yield MockSnoozClient(BLEDevice("Snooz-ABCD", "00:00:00:00:12:34"))
+    yield MockSnoozClient(BLEDevice("Snooz-ABCD", "00:00:00:00:12:34", [], 0))
 
 
 @pytest.fixture()

--- a/tests/test_device.py
+++ b/tests/test_device.py
@@ -94,7 +94,7 @@ class SnoozTestFixture:
 
 @pytest.fixture(scope="function")
 def snooz(mocker: MockerFixture, event_loop: AbstractEventLoop) -> SnoozTestFixture:
-    device = BLEDevice("AA:BB:CC:DD:EE:FF", "Snooz-EEFF")
+    device = BLEDevice("AA:BB:CC:DD:EE:FF", "Snooz-EEFF", [], 0)
     token = "AABBCCDDEEFF"
 
     def get_connected_client(

